### PR TITLE
Feature clubs by location

### PR DESCRIPTION
**Pull request by KattyaCuevas**
References #17 

It uses [Open Cage Data](https://opencagedata.com/) to transform adresses to coordinates.
It adds `geolib` to the pacakge json to calculate distances between coordinates.
It sorts all the clubs by their location.

### CODE REVIEW

File: client/src/services/geocode.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    api_url +
+    "?" +
+    "q=" +
+    encodeURIComponent(place) +
+    "&key=" +
+    apikey +
+    "&language=es" +
+    "&pretty=1" +
+    "&countrycode=pe" +
+    "&no_annotations=1";
```
Comment (liannivin) [inline client/src/services/geocode.js]: Could you please explain what this `piece of code` does?

File: client/src/views/home.js [hunk 2]
```diff
@@ -1,31 +1,130 @@
...
     <div>
-      <h2>Filters</h2>
-      {loading && <div>Loading</div>}
-      {clubs.map(club => (
-        <Club club={club} key={club.id} />
-      ))}
+      <div css={styleSelectsContainer}>
+        <Select
+          onChange={handleChangeLocation}
+          defaultValue=""
```
Comment (condef5) [inline client/src/views/home.js]: Could you assign a more significant name instead of `defaultValue` ?

Comment (cristianbgp) [reply client/src/views/home.js]: That's the default value for the select tag

File: client/src/services/geocode.js (hunk 1 above)
Comment (cristianbgp) [reply client/src/services/geocode.js]: That is the way to get a request from [Open Cage Data API](https://opencagedata.com/api)

**Pull request merged**